### PR TITLE
refactor(jira)!: rename myissues to mine and add completion

### PR DESCRIPTION
**Pull request by augustfengd**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- [...]

## Other comments:

Hi! I saw that there was missing an auto-completion for `myissues` so I added one. 

I also feel that `jira mine` feels more UX-y. If this is undesired I can revert the naming.

### CODE REVIEW

File: plugins/jira/README.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ This plugin supplies one command, `jira`, through which all its features are exp
 jira            # performs the default action
 
 jira new        # opens a new issue
+jira mine       # queries for you own issues
```
Comment (jmeridth) [inline plugins/jira/README.md:21]: `your` instead of `you`

**Review by jmeridth — COMMENTED** [review]
LGTM just one suggestion

File: plugins/jira/README.md (hunk 1 above)
Comment (augustfengd) [reply plugins/jira/README.md]: thanks! I just amended the commit and force pushed, hope that's alright.

**Review by jmeridth — APPROVED** [review]

**Comment by jmeridth** [comment]
@shvimas @mcornella you both seems to have merged recently on this plugin.  can you please review?  Thank you.

**Review by robbyrussell — APPROVED** [review]

**Comment by robbyrussell** [comment]
These look like solid recommendations. Merging!

**Pull request merged**